src/main/scala/com/yotpo/metorikku/configuration/job/input/Kafka.scala

```diff
@@ -10,10 +10,13 @@ case class Kafka(servers: Seq[String],
                  consumerGroup: Option[String],
                  options: Option[Map[String, String]],
                  schemaRegistryUrl:  Option[String],
-                 schemaSubject:  Option[String]
+                 schemaSubject:  Option[String],
+                 registeredSchemaId: Option[String],
```

Comment:
this is weird... all 3 say the same thing, that's what schemasubject was meant for, to change the schema ID to something else...